src/components/navigation/nav.module.scss

```diff
@@ -0,0 +1,365 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,400i,500,500i,700');
+@import '../../../styles/variables.scss';
+
+.navbar {
+  font-family: 'Roboto', sans-serif;
+  font-size: 14px;
+  // color: #FFF;
+  color: #000;
+  box-sizing: border-box;
+} //global body styles
+
+.navbar *{
+  box-sizing: inherit;
+}//global box-sizing
+
+::-webkit-scrollbar {
+  display: none;
+  }
+  i{
+    font-size: 18px;
+    cursor: pointer;
+  }
+
+.Link{
+  color: black;
+} 
+ 
+a {
+  text-decoration: none;
+  cursor: pointer;
+} //global link styles
+.vectors{
+  display:flex;
+  align-items: center;
+  // justify-content: center;
+}
+.right_nav{
+  nav{
+    display: none;
+  }
+}
+.top__bar {
+  z-index: 20;
+  position: relative;
+  width: 100%;
+  background-color: #fcfcfc;
+  height: 48px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.228);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0px 10px;
+
+  .logo__icon {
+    width: 10%;
+  }
+  .logo__text { 
+    height: 18px;
+  }
+  nav{
+      ul{
+         list-style: none;
+      }
+
+      li{
+          display: inline-block;
+          padding: 10px 40px;
+          transition: all 0.4s ease 0s;
+          font-size: 14px;
+          cursor: pointer;
+          &:hover{
+            background-color: $yellow;
+          }
+      }
+  }
+  .bars{
+      display: none;
+  }
+  input{
+      padding: 8px 25px;
+      border-radius: 5px;
+      border: none;
+      outline: none;
+      background-color: #9999996f;
```

Comment:
use variable, there are such colors, if not there,  just add it there as a variable as it will be used in by others